backend/kafka-queue/types.go

```diff
@@ -22,7 +22,6 @@ const (
 	AddSessionProperties             PayloadType = iota
 	PushBackendPayload               PayloadType = iota
 	PushMetrics                      PayloadType = iota
-	MarkBackendSetup                 PayloadType = iota
```

Comment:
we'll want to keep this here and add a note about it being deprecated. otherwise messages currently in flight will be interpreted incorrectly, as these `iota` values will be offset by 1 from the values in the current kafka messages

```suggestion
	// Deprecated: setup events are written from other payload processing
	MarkBackendSetup                 PayloadType = iota
```

Reply:
Ahh, yes, I meant to ask about this because I saw another note about a deprecated message type. Thanks for flagging.